src/left-nav.jsx

```diff
@@ -225,6 +236,28 @@ const LeftNav = React.createClass({
       </div>
     );
   },
+  
+  _isOpen() {
```

Comment:
I feel like `_isOpen` is adding complexity where it's not needed. Can we kill this property and rely on the `this.state.open`?